staging/src/k8s.io/kms/internal/plugins/_mock/go.sum

```diff
@@ -0,0 +1,66 @@
+github.com/ThalesIgnite/crypto11 v1.2.5 h1:1IiIIEqYmBvUYFeMnHqRft4bwf/O36jryEUpY+9ef8E=
```

Comment:
need to wait for @dims to restore the incorrectly removed file from https://github.com/kubernetes/kubernetes/pull/121338#pullrequestreview-1698718944, then rebase. sorry for the churn